shared/bls/blst/stub.go

```diff
@@ -1,4 +1,4 @@
-// +build darwin windows !blst_enabled
+// +build windows !blst_enabled
 
 package blst
```

Comment:
https://github.com/prysmaticlabs/prysm/blob/e5992a58ea4bc63223a460bacd7c552069bf7c08/shared/bls/blst/stub.go#L9-L10

these line should also be fixed